bin/cron/hoc_signup_counts

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+# This script performs aggregation queries on the pegasus.forms table, specifically HOC signup
+# forms, saving the results to the pegasus.properties table.
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/db'
+require 'cdo/only_one'
+require 'cdo/properties'
+require 'dynamic_config/dcdo'
+
+DEFAULT_YEAR = 2017
+PEGASUS_REPORTING_DB_READONLY = sequel_connect(
+  CDO.pegasus_reporting_db_reader,
+  CDO.pegasus_reporting_db_reader
+)
+
+def main
+  year = (ARGV[0] || DCDO.get('hoc_year', DEFAULT_YEAR)).to_i
+
+  # Note that country_counts and us_state_counts are of the form
+  # [{"country" => nil, "count" => 1}, {"country" => "Afghanistan", "count" => 2}, ...].
```

Comment:
Curious.. what does country of `nil` mean?  And what does a US state one look like?